### PR TITLE
Updated database location for Incident Report project

### PR DESCRIPTION
**Pull request by matthamil**
## Status
**READY**

## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[X] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

## Resolves Issue Number

https://github.com/TeamCharles/customer-service/issues/45

## Deploy Notes

```bash
git pull --prune
git checkout new-db-variable
rm bangazon.db
dotnet ef database update
dotnet run
```

## Steps to Test or Reproduce
Update your environment variable pointing to your database to `Bangazon_Workforce_DB`

1. Nuke your database

1. Rebuild it.

1. Make a new employee

1. Try to make him/her an Admin

## Impacted Files in Application

* README.md

* Startup.cs

### CODE REVIEW

**Comment by matthamil** [comment]
@TeamCharles/team Please review the updated README and `Startup.cs`

**Comment by garrett-vangilder** [comment]
LGTM

**Comment by daynewright** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ export ASPNETCORE_ENVIRONMENT="Development"
 
 On initial installation of the Banagazon web application you must set an environment variable to your local database. (This database with the file name bangazon.db will be created when you run dotnet ef database update later in the installation process.)
 ```Bash
-export BangazonWeb_Db_Path="/path/to/bangazon.db"
+export BangazonWeb_Db_Path="/path/to/bangazon_workforce.db"
```
Comment (MattKraatz) [inline README.md]: variable should be named `Bangazon_Workforce_DB`

File: README.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ $env:ASPNETCORE_ENVIRONMENT="Development"
 
 On initial installation of the Banagazon web application you must set an environment variable to your local database. Even though you have not yet created a database, this variable will set the location of your database for later after creation.
 ```Bash
-$env:BangazonWeb_Db_Path="/path/to/bangazon.db"
+$env:BangazonWeb_Db_Path="/path/to/bangazon_workforce.db"
```
Comment (MattKraatz) [inline README.md]: variable should be named `Bangazon_Workforce_DB`

File: src/workforce-management/Views/Employee/Add.cshtml [hunk 3]
```diff
@@ -55,6 +55,15 @@
                 @Html.ListBoxFor(m => m.EnrolledTraining, new MultiSelectList(Model.TrainingPrograms, "TrainingProgramId", "Name", null), new { @class = "form-control" })
             </div>
         </div>
+
+        <div class="form-group">
+            <label asp-for="Employee.Administrator" class="col-md-2 control-label">Administrator?</label>
+            <div class="col-md-10">
+                <input type="checkbox" asp-for="Employee.Administrator" class="form-control" />
```
Comment (MattKraatz) [inline src/workforce-management/Views/Employee/Add.cshtml:62]: Beautiful checkbox

**Review by MattKraatz — APPROVED** [review]
LGTM. Quick change on the Readme

**Pull request merged**